examples/flows/evaluation/eval-chat-math/line_process.py

```diff
@@ -0,0 +1,55 @@
+from promptflow import tool
+
+def string_to_number(raw_string: str, processed_result: int) -> list:
+    ''' Try to parse the prediction string and groundtruth string to float number. 
+    Support parse int, float, fraction and recognize non-numeric string with wrong format.
+    For example: '3/5', '6/10', '0.5', 'the answer is \box{2/3}', '4/7//8'

```

Comment:
these cases should not be able to parse one? If so, better to mention to clarify these're the wrong format case

Reply:
Thanks, that will be better. Have updated.